controllers/datadogagent/feature/liveprocess/feature.go

```diff
@@ -115,6 +168,16 @@ func (f *liveProcessFeature) manageNodeAgent(agentContainerName apicommonv1.Agen
 	managers.VolumeMount().AddVolumeMountToContainer(&passwdVolMount, agentContainerName)
 	managers.Volume().AddVolume(&passwdVol)
 
+	// cgroups volume mount
+	cgroupsVol, cgroupsVolMount := volume.GetVolumes(apicommon.CgroupsVolumeName, apicommon.CgroupsHostPath, apicommon.CgroupsMountPath, true)
+	managers.VolumeMount().AddVolumeMountToContainer(&cgroupsVolMount, agentContainerName)
+	managers.Volume().AddVolume(&cgroupsVol)
+
+	// procdir volume mount
+	procdirVol, procdirVolMount := volume.GetVolumes(apicommon.ProcdirVolumeName, apicommon.ProcdirHostPath, apicommon.ProcdirMountPath, true)
+	managers.VolumeMount().AddVolumeMountToContainer(&procdirVolMount, agentContainerName)
+	managers.Volume().AddVolume(&procdirVol)
```

Comment:
Should we gate these mounts behind the `runInCoreAgent` enablement?

Reply:
I added these here because they are required for the feature to work on any container when live containers is disabled explicitly (its usually enabled by default). It's a rare occurrence, but this allows the feature to be independent.